python/ray/actor.py

```diff
@@ -779,24 +792,26 @@ def _serialization_helper(self, ray_forking):
         """
         worker = ray.worker.get_global_worker()
         worker.check_connected()
-        state = {
-            "actor_language": self._ray_actor_language,
-            # Local mode just uses the actor ID.
-            "core_handle": worker.core_worker.serialize_actor_handle(
-                self._ray_actor_id)
-            if hasattr(worker, "core_worker") else self._ray_actor_id,
-            "method_decorators": self._ray_method_decorators,
-            "method_signatures": self._ray_method_signatures,
-            "method_num_return_vals": self._ray_method_num_return_vals,
-            "actor_method_cpus": self._ray_actor_method_cpus,
-            "is_cross_language": self._ray_is_cross_language,
-            "actor_creation_function_descriptor": self.
-            _ray_actor_creation_function_descriptor,
-        }
+
+        if hasattr(worker, "core_worker"):
+            state = worker.core_worker.serialize_actor_handle(self)
+        else:
```

Comment:
document that the if block is non-local mode, and the else block is local mode.

Reply:
Updated.